modules/apps/27-interchain-accounts/controller/types/msgs.go

```diff
@@ -46,3 +48,24 @@ func (msg MsgRegisterAccount) GetSigners() []sdk.AccAddress {
 
 	return []sdk.AccAddress{accAddr}
 }
+
+// NewMsgSubmitTx creates a new instance of MsgSubmitTx
+func NewMsgSubmitTx(connectionID, owner string, timeoutHeight clienttypes.Height, timeoutTimestamp uint64, msgs []*icatypes.InterchainAccountPacketData) *MsgSubmitTx {
+	return &MsgSubmitTx{
+		ConnectionId:     connectionID,
+		Owner:            owner,
+		TimeoutHeight:    timeoutHeight,
+		TimeoutTimestamp: timeoutTimestamp,
+		Msgs:             msgs,
+	}
+}
+
+// ValidateBasic implements sdk.Msg
+func (msg MsgSubmitTx) ValidateBasic() error {
+	return nil
+}
+
+// GetSigners implements sdk.Msg
+func (msg MsgSubmitTx) GetSigners() []sdk.AccAddress {
+	return []sdk.AccAddress{}
```

Comment:
```suggestion
	return []sdk.AccAddress{owner}
```

This needs to be authenticated by owner signature

Reply:
yea i would fill out this function in the next PR along with validate basic :)